plugins/examples/webrtcsink-stats/src/App.svelte

```diff
@@ -80,7 +80,14 @@
   }
 
   const fetchStats = () => {
-    ws = new WebSocket("ws://127.0.0.1:8484");
+    const urlParams = new URLSearchParams(window.location.search);
```

Comment:
can you document that behavior in https://github.com/centricular/webrtcsink/blob/main/plugins/examples/webrtcsink-stats/README.md?